src/client/components/common/Error.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Link } from 'react-router-dom';
+import { Grid, makeStyles, Typography } from '@material-ui/core';
+import { HOME_ROUTE } from '../../config/routes';
+
+const useStyles = makeStyles((theme) => ({
+  link: {
+    display: 'block',
+    margin: 'auto',
+    marginTop: theme.spacing(2),
+  },
+  content: {
+    textAlign: 'center',
+  },
+}));
+
+const Error = () => {
```

Comment:
This seems to be specifically a 404 error. Do we want to generalise by passing the string as a prop? Or have a switch? Or separate components?